src/electron/child_process.ts

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { ChildProcess, spawn } from 'child_process';
+
+// Simple child process launcher.
+//
+// NOTE: Because there is no way in Node.js to tell whether a process launched successfully,
+//       #startInternal always succeeds; use #onExit to be notified when the process has exited
```

Comment:
It looks like `startInternal` has been renamed?

Reply:
Done. Updated the documentation to reflect the new behavior.